cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/DataprocJobMain.java

```diff
@@ -69,6 +69,10 @@ public static void main(String[] args) throws Exception {
       throw new RuntimeException(
           "Missing --" + SPARK_COMPAT + " argument for the spark compat version");
     }
+    if (!arguments.containsKey(Constants.Files.APPLICATION_JAR)) {
+      throw new RuntimeException(
+        "Missing --" + Constants.Files.APPLICATION_JAR + " argument for the application jar name");
```

Comment:
Use String.format

Reply:
Why not to assume Constants.Files.APPLICATION_JAR if it's not provided in args?

Reply:
Basically we add the localized name of APPLICATION_JAR in dataproc job runtime arguments like `application.jar=APPLICATION_JAR_LOCALIZED_NAME` which is required to know the name of jar to add it in classpath.

Reply:
Discussed offline.